rtv/submission.py

```diff
@@ -171,6 +171,8 @@ def draw_submission(win, data):
         attr = curses.A_BOLD | Color.GREEN
         text = '{author}'.format(**data)
         win.addnstr(row, 1, text, n_cols, attr)
+        text = '{flair}'.format(**data)
+        win.addnstr(row, 1, text, n_cols, curses.A_BOLD | Color.GREEN)
```

Comment:
This will overwrite the author name, as the text for both lines starts on the same row and column. If you want the text to have the same formatting as the author, I would suggest doing what you did above e.g. ``'{author} {flair}'``. If you want the flair to have different text formatting as the author, you can copy what I did below for the ``'{created} {subreddit}'`` text.